cli-release/releases_distribution_rule.json

```diff
@@ -0,0 +1,9 @@
+{
+  "distribution_rules": [
+    {
+      "site_name": "*",
+      "city_name": "*",
+      "country_codes": ["*"]
+    }
+  ]
+}
```

Comment:
Let's remove this file. We shouldn't distribute to all sites and e can use `--site` flag instead of providing distribution rules.
```suggestion
```